pubmed_dl/main.py

```diff
@@ -1,15 +1,16 @@
 import json
+import typer
 from pubmed_dl.ncbi import get_list_pmid, uids_to_docs
 
-def main():
-    start = input("Enter start date (YYYY/MM/DD): ")
-    end = input("Enter end date (YYYY/MM/DD): ")
-    val = get_list_pmid(start,end)
+def main(start: str, end: str):
+    start_date = start
+    end_date = end
+    val = get_list_pmid(start_date,end_date)
     with open("test.json", "w") as f:

```

Comment:
`"test.json"` is hardcoded. It should be one of the arguments to `main` that a user can specify at the command line. Remember that if they specify a directory you will have to create it. I use the following code typically:

```python
from pathlib import Path

# output_filepath provided as command line argument as string...

output_filepath: Path = Path(output_filepath)
output_filepath.parents[0].mkdir(parents=True, exist_ok=True)
```